src/bandersnatch/package.py

```diff
@@ -204,6 +205,15 @@ def sync_release_files(self):
 
         self.mirror.altered_packages[self.name] = downloaded_files
 
+    def gen_data_requires_python(self, release):
+        if "requires_python" in release and release["requires_python"] is not None:
```

Comment:
It will never be None - Seems it will only be "" ... so just:
```
and release["requires_python"]
```
- Would probably do, but this will work.